model/github_runner.rb

```diff
@@ -23,6 +25,22 @@ def runner_url
     "http://github.com/#{repository_name}/settings/actions/runners/#{runner_id}" if runner_id
   end
 
+  def init_health_monitor_session
+    {
+      ssh_session: vm.sshable.start_fresh_session
+    }
+  end
+
+  def check_pulse(session:, previous_pulse:)
+    reading = begin
+      available_memory = session[:ssh_session].exec!("free | awk 'NR==2 {print $3}'").chomp
```

Comment:
Is it available memory or used memory?

```shell
ubi@vm83n0bh:~$ free
               total        used        free      shared  buff/cache   available
Mem:         8125000      201984     6704188         888     1218828     7665308
Swap:              0           0           0
ubi@vm83n0bh:~$ free | awk 'NR==2 {print $3}'
201984
```

Reply:
Good catch. Fixed it.

Reply:
should we print both to make it easier to understand the situation? Depending on the VM size, the free memory size may mean different.

Reply:
That information would be useful only once, (i.e. once you see one used/available pair, you wouldn't need to see another pair, you can deduce the one from the other). Collecting it in every 5 seconds seems like a waster. Also we can get the total memory amount from other sources like database. In the interest of not doing lots of things in the monitor, I'd like to skip doing that.